File: Group16/Blood Donation/app/src/main/java/com/example/blooddrop/MainActivity.java
package com.example.blooddrop;

import android.content.Intent;
import android.os.Bundle;
import android.widget.Button;
import androidx.appcompat.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {

    Button btnUser, btnAdmin;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        btnUser = findViewById(R.id.btnUser);
        btnAdmin = findViewById(R.id.btnAdmin);

        // 🩸 USER → Open UserChoiceActivity
        btnUser.setOnClickListener(v -> {
            startActivity(new Intent(MainActivity.this,
                    UserChoiceActivity.class));
        });

        // 🔐 ADMIN → Open AdminLoginActivity
        btnAdmin.setOnClickListener(v -> {
            startActivity(new Intent(MainActivity.this,
                    AdminLoginActivity.class));
        });
    }
}

File: Group 17/Lendloop2/app/src/main/java/com/example/lendloop2/MainActivity.kt
package com.example.lendloop2

import android.content.Intent
import android.os.Bundle
import android.os.Handler
import android.os.Looper
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        Handler(Looper.getMainLooper()).postDelayed({
            if (!isFinishing && !isDestroyed) {
                startActivity(Intent(this, LoginActivity::class.java))
                finish()
            }
        }, 2000)
    }
}